java/client/src/main/java/glide/api/models/BaseTransaction.java

```diff
@@ -1451,13 +1450,13 @@ public T zscore(@NonNull String key, @NonNull String member) {
      *     <code>null</code> will be returned.
      */
     public T zrank(@NonNull String key, @NonNull String member) {
-        ArgsArray commandArgs = buildArgs(key, member);
+        ArgsArray commandArgs = buildArgs(new String[] {key, member});
         protobufTransaction.addCommands(buildCommand(Zrank, commandArgs));
         return getThis();
     }
 
     /**
-     * Returns the rank of <code>member</code> in the sorted set stored at <code>key</code> with its
+     * Returns the rank of <code>member</code> in the sorted set stored at <code>key</code> with it's
```

Comment:
You had it right the first time. Possessive is "its".

Reply:
I moved this to another PR #201 and reverted here